Transmission.xcodeproj/project.pbxproj

```diff
@@ -2207,7 +2207,7 @@
 				11524394C75E57E52CD9ADF2 /* dns-ev.h in Headers */,
 				C1077A4F183EB29600634C22 /* error.h in Headers */,
 				A2679295130E00A000CB7464 /* tr-utp.h in Headers */,
-				A2679295130E00A000CB7464 /* tr-buffer.h in Headers */,
```

Comment:
Here you can see what was the issue: both tr-utp.h and tr-buffer.h had ID A2679295130E00A000CB7464.

Reply:
d'oh